carbon-mb/components/transports/mqtt/src/main/java/org/wso2/carbon/andes/transports/mqtt/adaptors/andes/utils/MqttUtils.java

```diff
@@ -300,4 +195,8 @@ public static boolean isDurable(boolean cleanSession, int qos) {
         return durable;
     }
 
+    public static ProtocolType getProtocolType() {
```

Comment:
Similar to AMQP I've used the MQTTChannel to set the proper protocol version for both messages and subscriptions. 

Updated the subscription store registration as well to have both MQTT version support